lib/plugins/aws/deploy/lib/ensure-valid-bucket-exists.js

```diff
@@ -31,7 +31,7 @@ module.exports = {
         });
       } catch (err) {
         throw new ServerlessError(
-          `Could not locate deployment bucket. Error: ${err.message}`,
+          `Could not locate deployment bucket: ${this.bucketName}. Error: ${err.message}`,
```

Comment:
Let's also wrap bucket name in parenthesis (it's our convention for adding names to error messages)

Reply:
@medikoo alright, the patch has been updated, thanks!